internal/cli/atlas/dbusers/describe.go

```diff
@@ -95,7 +92,11 @@ func DescribeBuilder() *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVar(&opts.authDB, flag.AuthDB, convert.AdminDB, usage.AuthDB)
+	if config.BinName() == config.AtlasCLI {
+		cmd.Flags().StringVar(&opts.authDB, flag.AuthDB, convert.AdminDB, usage.AtlasAuthDB)
+	} else {
+		cmd.Flags().StringVar(&opts.authDB, flag.AuthDB, convert.AdminDB, usage.AuthDB)
+	}
```

Comment:
there's no need for this `if` you can safety use `usage.AtlasAuthDB` with no major impact, the issue was with `usage.AuthDB` being reused for both ops manager and atlas commands, this is not about mongocli vs atlas cli

Reply:
Oh ok, I will change it